### DISCUSSION TITLE
Support Confluence Cloud for Bitbucket.org Dependency Dashboards

### DISCUSSION THREAD

**Discussion by setchy** [renovatebot/renovate#24831] [Suggest an Idea]
### What would you like Renovate to be able to do?

For Bitbucket Cloud users, support integration with Confluence Cloud for publishing dependency dashboard(s)

_Originally suggested in https://github.com/renovatebot/renovate/issues/16625_

### If you have any ideas on how this should be implemented, please tell us here.

Creating and updating the wiki pages will be relatively straight-forward.  

Linking repositories to specific wiki spaces will require some further thought.

Alternatively, perhaps a single wiki space could be configured as a self-hosted configuration option, which would simplify the above.

### Is this a feature you are interested in implementing yourself?

Maybe

**Comment by kwelch** [comment]
I am interested in working on this as well. I have some code to publish to confluence already in our internal Atlassian instance, but I need to clean it up to fit in this project better.

**Comment by setchy** [comment]
cross-ref: first-class Bitbucket Cloud Issues support is being sunset mid August (see https://github.com/renovatebot/renovate/pull/42633 and [official announcement](https://community.atlassian.com/forums/Bitbucket-articles/Announcing-sunset-of-Bitbucket-Issues-and-Wikis/ba-p/3193882))
